src/function.rs

```diff
@@ -136,3 +141,42 @@ fn just_executable(_context: &FunctionContext) -> Result<String, String> {
     )
   })
 }
+
+fn file_name(_context: &FunctionContext, file_path: &str) -> Result<String, String> {
```

Comment:
For the argument name for this and the other functions,, how about `path` instead?